UI/JustAssembly/App.xaml.cs

```diff
@@ -9,6 +9,8 @@ namespace JustAssembly
 {
     public partial class App : Application
     {
+        private string[] _args;
```

Comment:
Using underscores in from of names marking the member as private is against our coding practices. I suggest renaming all fields to `args`.

Reply:
Fixed. I'm fighting my IDE a bit because my current style is different from yours, so let me know if you catch any other formatting/style issue (I just fixed a few brackets that were in OTB style)